autogpt_platform/backend/backend/server/rest_api.py

```diff
@@ -137,10 +138,15 @@ def run(self):
             allow_methods=["*"],  # Allows all methods
             allow_headers=["*"],  # Allows all headers
         )
+        log_config = dict(uvicorn.config.LOGGING_CONFIG)
+        log_config["loggers"]["uvicorn"] = {"handlers": []}
+        log_config["loggers"]["uvicorn.error"] = {"handlers": []}
+        log_config["loggers"]["uvicorn.access"] = {"handlers": []}
         uvicorn.run(
             server_app,
             host=backend.util.settings.Config().agent_api_host,
             port=backend.util.settings.Config().agent_api_port,
+            log_config=log_config,
```

Comment:
IIRC you can achieve all of this with just `log_config=None`. I did that a while ago and am not sure when it was unfixed.
```suggestion
        uvicorn.run(
            server_app,
            host=backend.util.settings.Config().agent_api_host,
            port=backend.util.settings.Config().agent_api_port,
            log_config=None,
```

Reply:
it was removed in this pr 
https://github.com/Significant-Gravitas/AutoGPT/pull/8519/files

Reply:
those aren't the same behavior 

Setting to None removes the logging config and depends on yours entirely

I want uvicorn's configs but my handlers for its sinks. Below is the default, I'm setting each of the sinks to use our handlers

```py
LOGGING_CONFIG: dict[str, Any] = {
    "version": 1,
    "disable_existing_loggers": False,
    "formatters": {
        "default": {
            "()": "uvicorn.logging.DefaultFormatter",
            "fmt": "%(levelprefix)s %(message)s",
            "use_colors": None,
        },
        "access": {
            "()": "uvicorn.logging.AccessFormatter",
            "fmt": '%(levelprefix)s %(client_addr)s - "%(request_line)s" %(status_code)s',  # noqa: E501
        },
    },
    "handlers": {
        "default": {
            "formatter": "default",
            "class": "logging.StreamHandler",
            "stream": "ext://sys.stderr",
        },
        "access": {
            "formatter": "access",
            "class": "logging.StreamHandler",
            "stream": "ext://sys.stdout",
        },
    },
    "loggers": {
        "uvicorn": {"handlers": ["default"], "level": "INFO", "propagate": False},
        "uvicorn.error": {"level": "INFO"},
        "uvicorn.access": {"handlers": ["access"], "level": "INFO", "propagate": False},
    },
}
```

Reply:
Don't we also want uvicorn to use the same logging format as the rest of the app?